client/reader/sidebar/reader-sidebar-lists/list.jsx

```diff
@@ -1,46 +1,33 @@
-import { isEnabled } from '@automattic/calypso-config';
-import { map } from 'lodash';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
 import ListItem from './list-item';
 import ListItemCreateLink from './list-item-create-link';
 
-export class ReaderSidebarListsList extends Component {
+export default class ReaderSidebarListsList extends Component {
 	static propTypes = {
 		lists: PropTypes.array,
 		path: PropTypes.string.isRequired,
 		currentListOwner: PropTypes.string,
 		currentListSlug: PropTypes.string,
-		translate: PropTypes.func,
 	};
 
-	renderItems() {
-		const { currentListOwner, currentListSlug, path } = this.props;
-		return map( this.props.lists, ( list ) => {
-			return (
-				<ListItem
-					key={ list.ID }
-					list={ list }
-					path={ path }
-					currentListOwner={ currentListOwner }
-					currentListSlug={ currentListSlug }
-				/>
-			);
-		} );
-	}
-
 	render() {
+		const { lists, currentListOwner, currentListSlug, path } = this.props;
 		/* eslint-disable wpcalypso/jsx-classname-namespace */
 		return (
 			<ul className="sidebar__menu-list">
-				{ this.renderItems() }
-				{ isEnabled( 'reader/list-management' ) && (
-					<ListItemCreateLink key="create-item-link" path={ this.props.path } />
-				) }
+				{ lists.map( ( list ) => (
+					<ListItem
+						key={ list.ID }
+						list={ list }
+						path={ path }
+						currentListOwner={ currentListOwner }
+						currentListSlug={ currentListSlug }
+					/>
+				) ) }
+				<ListItemCreateLink key="create-item-link" path={ path } />
```

Comment:
In this component I did a few drive-by changes that go beyond removing a feature flag:
- inline the `renderItems` method into the `render` method
- replace Lodash `map` with native array `map`. The `lists` prop is guaranteed to always be an array because it comes from the `getSubscribedLists` selector. We don't need to worry about a `null` value.
- remove the unused `translate` prop that is not even really passed to the component
- remove the non-default export that is not used

I could also convert the component to functional, remove proptypes and migrate to TypeScript. But I didn't want to fall into a rabbit hole 🙂